elements/reigns/src/features/game/validateGameDefinition.ts

```
@@ -4,9 +4,49 @@ export const validateGameDefinition = (
   definition: GameDefinition
 ): GameDefinition => ({
   ...definition,
+  assetsUrl: validateAssetsUrl(definition.assetsUrl),
   ...validateCards(definition.cards),
 });
 
+const urlWithoutTrailingSlash = (url: string) => {
+  if (!url) return "";
+
+  if (url.slice(-1) !== "/") {
+    return url;
+  }
+
+  return url.slice(0, -1);
+};
+
+export const validateAssetsUrl = (url: string) => {
```

i would expect `validate*` to throw (or otherwise mark invalid) on invalid input, rather than mutate, so perhaps a rename to `getAssetsUrl` or `parseAssetsUrl`  or `mapAssetsUrl` or similar.